### PR TITLE
[debug]debug with profiler

### PR DESCRIPTION
**Pull request by dzhwinter**

### CODE REVIEW

File: fluid/vgg16.py [hunk 1]
```diff
@@ -94,23 +97,24 @@ def main():
     # Evaluator
     accuracy = fluid.evaluator.Accuracy(input=predict, label=label)
```
Comment (ranqiu92) [inline fluid/vgg16.py]: fluid.evaluator 下无 Accuracy，请针对本文件所有相应地方做相应修改。

File: fluid/resnet50.py [hunk 2]
```diff
@@ -190,7 +190,7 @@ def run_benchmark(model, args):
     opts = optimizer.minimize(avg_cost)
     accuracy = fluid.evaluator.Accuracy(input=predict, label=label)
```
Comment (ranqiu92) [inline fluid/resnet50.py]: fluid.evaluator 下无 Accuracy，请针对本文件所有相应地方做相应修改。
以及前面的  paddle.v2.fluid -> paddle.fluid

File: fluid/mnist.py [hunk 3]
```diff
@@ -141,37 +141,39 @@ def run_benchmark(model, args):
...
-
-        train_avg_acc = accuracy.eval(exe)
-        test_avg_acc = eval_test(exe, accuracy, inference_program)
-
-        print("pass=%d, train_avg_acc=%f, test_avg_acc=%f, elapse=%f" %
-              (pass_id, train_avg_acc, test_avg_acc,
-               (pass_end - pass_start) / 1000))
+    with profiler.profiler(args.device, 'total') as prof:
+        for pass_id in range(args.pass_num):
+            accuracy.reset(exe)
```
Comment (ranqiu92) [inline fluid/mnist.py]: fluid.evaluator 下无 Accuracy，请针对本文件所有相应地方做相应修改。
以及前面的 paddle.v2.fluid -> paddle.fluid

File: fluid/mnist.py [hunk 4]
```diff
@@ -141,37 +141,39 @@ def run_benchmark(model, args):
...
+                    feed={"pixel": img_data,
+                          "label": y_data},
+                    fetch_list=[avg_cost] + accuracy.metrics
+                )  # The accuracy is the accumulation of batches, but not the current batch.
+
+                end = time.time()
+                loss = np.array(outs[0])
+                acc = np.array(outs[1])
+                print("pass=%d, batch=%d, loss=%f, error=%f, elapse=%f" %
+                      (pass_id, batch_id, loss, 1 - acc, (end - start) / 1000))
```
Comment (ranqiu92) [inline fluid/mnist.py]: 计时建议带上单位，以及  这里是要以 ks 为单位吗？
（下同）
